xlators/protocol/server/src/server.h

```diff
@@ -55,7 +54,6 @@ struct _child_status {
 };
 struct server_conf {
```

Comment:
nit: if you are already making these changes, would be nice to properly align the struct to reduce padding and improve cacheline locality. I usually use pahole utility for such work:
```
struct server_conf {
        rpcsvc_t *                 rpc;                  /*     0     8 */
        struct rpcsvc_config       rpc_conf;             /*     8     4 */
        int                        inode_lru_limit;      /*    12     4 */
        _Bool                      verify_volfile;       /*    16     1 */
        _Bool                      trace;                /*    17     1 */

        /* XXX 6 bytes hole, try to pack */

        char *                     conf_dir;             /*    24     8 */
        struct _volfile_ctx *      volfile;              /*    32     8 */
        dict_t *                   auth_modules;         /*    40     8 */
        pthread_mutex_t            mutex;                /*    48    40 */
        /* --- cacheline 1 boundary (64 bytes) was 24 bytes ago --- */
        struct list_head           xprt_list;            /*    88    16 */
        pthread_t                  barrier_th;           /*   104     8 */
        _Bool                      server_manage_gids;   /*   112     1 */

        /* XXX 7 bytes hole, try to pack */

        gid_cache_t                gid_cache;            /*   120 49200 */
        /* --- cacheline 770 boundary (49280 bytes) was 40 bytes ago --- */
        int32_t                    gid_cache_timeout;    /* 49320     4 */
        int                        event_threads;        /* 49324     4 */
        _Bool                      parent_up;            /* 49328     1 */
        _Bool                      dync_auth;            /* 49329     1 */

        /* XXX 6 bytes hole, try to pack */

        struct _child_status *     child_status;         /* 49336     8 */
        /* --- cacheline 771 boundary (49344 bytes) --- */
        gf_lock_t                  itable_lock;          /* 49344    40 */
        _Bool                      strict_auth_enabled;  /* 49384     1 */

        /* size: 49392, cachelines: 772, members: 20 */
        /* sum members: 49366, holes: 3, sum holes: 19 */
        /* padding: 7 */
        /* last cacheline: 48 bytes */
```

Reply:
OK but 1) reordering struct/union fields makes the patches more confusing for the reader and 2) cache line alignment issues yielding in performance penalties are usually meaningful only on really critical code paths.